spec/promoted/ruby/client_spec.rb

```diff
@@ -429,7 +429,7 @@
       expect(deliver_resp[:log_request]).to be nil
     end
 
-    it "delivers with nil insertions, which should default to request insertions" do
+    it "delivers with nil insertions, which is not an error" do
```

Comment:
Looking at it now, I think it should be an error, but we can save that for another day.

Reply:
Oh okay.  I'll hold off on merging this.  We can chat about it next week.